apertium-eng.eng.dix

```diff
@@ -65069,133 +65079,412 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+<e lm="Solingen">        <i>Solingen</i><par n="Barcelona__np"/></e>
+<e lm="Solnechnogorsk">  <i>Solnechnogorsk</i><par n="Barcelona__np"/></e>
+<e lm="Solntsevo">       <i>Solntsevo</i><par n="Barcelona__np"/></e>
 <e lm="Solomon Islands"> <i>Solomon<b/>Islands</i><par n="United_States__np"/></e>
 <e lm="Somalia">         <i>Somalia</i><par n="Barcelona__np"/></e>
 <e lm="Somaliland">      <i>Somaliland</i><par n="Barcelona__np"/></e>
 <e lm="Somerset">        <i>Somerset</i><par n="Barcelona__np"/></e>
+<e lm="Somerville">      <i>Somerville</i><par n="Barcelona__np"/></e>
+<e lm="Song Cau">        <i>Song<b/>Cau</i><par n="Barcelona__np"/></e>
+<e lm="Songjiang">       <i>Songjiang</i><par n="Barcelona__np"/></e>
+<e lm="Songyang">        <i>Songyang</i><par n="Barcelona__np"/></e>
+<e lm="Songyuan">        <i>Songyuan</i><par n="Barcelona__np"/></e>
+<e lm="Sonipat">         <i>Sonipat</i><par n="Barcelona__np"/></e>
+<e lm="Sopur">           <i>Sopur</i><par n="Barcelona__np"/></e>
+<e lm="Sorgun">          <i>Sorgun</i><par n="Barcelona__np"/></e>
 <e lm="Sosa">            <i>Sosa</i><par n="Barcelona__np"/></e>
+<e lm="Sougueur">        <i>Sougueur</i><par n="Barcelona__np"/></e>
+<e lm="Souk Ahras">      <i>Souk<b/>Ahras</i><par n="Barcelona__np"/></e>
+<e lm="Sousse">          <i>Sousse</i><par n="Barcelona__np"/></e>
+<e lm="south-western Iceland"><i>south-western<b/>Iceland</i><par n="Barcelona__np"/></e>
 <e lm="South Africa">    <i>South<b/>Africa</i><par n="Barcelona__np"/></e>
 <e lm="South America">   <i>South<b/>America</i><par n="Barcelona__np"/></e>
+<e lm="South Bend">      <i>South<b/>Bend</i><par n="Barcelona__np"/></e>
 <e lm="South Carolina">  <i>South<b/>Carolina</i><par n="Barcelona__np"/></e>
 <e lm="South Dakota">    <i>South<b/>Dakota</i><par n="Barcelona__np"/></e>
 <e lm="South Glamorgan"> <i>South<b/>Glamorgan</i><par n="Barcelona__np"/></e>
 <e lm="South Island">    <i>South<b/>Island</i><par n="Barcelona__np"/></e>
+<e lm="South Jordan">    <i>South<b/>Jordan</i><par n="Barcelona__np"/></e>
 <e lm="South Korea">     <i>South<b/>Korea</i><par n="Barcelona__np"/></e>
+<e lm="South Lawndale">  <i>South<b/>Lawndale</i><par n="Barcelona__np"/></e>
 <e lm="South Pole">      <i>South<b/>Pole</i><par n="Barcelona__np"/></e>
+<e lm="South San Francisco"><i>South<b/>San<b/>Francisco</i><par n="Barcelona__np"/></e>
 <e lm="South Sudan">     <i>South<b/>Sudan</i><par n="Barcelona__np"/></e>
 <e lm="South Wales">     <i>South<b/>Wales</i><par n="Barcelona__np"/></e>
 <e lm="Southampton">     <i>Southampton</i><par n="Barcelona__np"/></e>
 <e lm="Southern Bavaria"><i>Southern<b/>Bavaria</i><par n="Barcelona__np"/></e>
 <e lm="Southern Europe"> <par n="Ss"/><i>outhern<b/>Europe</i><par n="Barcelona__np"/></e>
-<e lm="South-Korea">     <i>South-Korea</i><par n="Barcelona__np"/></e>
-<e lm="south-western Iceland"><i>south-western<b/>Iceland</i><par n="Barcelona__np"/></e>
```

Comment:
Why were these removed?  This could break a released language pair.

Reply:
they are not removed i just removed the "-". is it make any changes??

Reply:
line 65439 and 65440 its still there but it says it removed.  and why is that?

Reply:
Why does it show that these lines were removed?

Reply:
i dont know too.

Reply:
I just added it before wait that must be the reason. alphabetical sorting error.
the symbols are after the letters :).

Reply:
*space

Reply:
Done :)